include/deal.II/fe/fe_abf.h

```diff
@@ -173,28 +173,6 @@ class FE_ABF : public FE_PolyTensor<PolynomialsABF<dim>, dim>
   void initialize_restriction ();
 
   /**
-   * Given a set of flags indicating what quantities are requested from a @p
-   * FEValues object, return which of these can be precomputed once and for
-   * all. Often, the values of shape function at quadrature points can be
-   * precomputed, for example, in which case the return value of this function
-   * would be the logical and of the input @p flags and @p update_values.
-   *
-   * For the present kind of finite element, this is exactly the case.
-   */
-  virtual UpdateFlags update_once (const UpdateFlags flags) const;
```

Comment:
Yes, exactly.. FE_PolyTensor is responsible for doing the computations and it knows the flags required.